app/screens/edit_profile/components/field.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {MessageDescriptor} from '@formatjs/intl/src/types';
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {KeyboardTypeOptions, Platform, TextInputProps, View} from 'react-native';
+
+import FloatingTextInput from '@components/floating_text_input_label';
+import {useTheme} from '@context/theme';
+import {getKeyboardAppearanceFromTheme, makeStyleSheetFromTheme} from '@utils/theme';
+
+import FieldDescription from './field_description';
+
+type FieldProps = TextInputProps & {
+    isDisabled?: boolean;
+    fieldDescription?: string;
+    key: string;
```

Comment:
Key won't work cause of is a reserve prop, what about `objKey`?

Reply:
Renamed to `fieldKey`